pkgs/development/python-modules/empyrical/default.nix

```diff
@@ -0,0 +1,43 @@
+{ stdenv
+, buildPythonPackage
+, fetchFromGitHub
+, isPy27
+, numpy
+, pandas
+, scipy
+, pandas-datareader
+, parameterized
+, python
+}:
+
+buildPythonPackage rec {
+  pname = "empyrical";
+  version = "0.5.5";
+  disabled = isPy27;
+
+  src = fetchFromGitHub {
+    owner = "quantopian";
+    repo = pname;
+    rev = version;
+    sha256 = "1cjplgh7kqqkav1l42r91h1s575xjv68s8awxyf5749wg1jwkfja";
+  };
+
+  # one test is known to fail: https://github.com/quantopian/empyrical/issues/131
+  patchPhase = ''
+    sed -i 's/    def test_perf_attrib_simple(self):/    @unittest.skip("known to fail")\n    def test_perf_attrib_simple(self):/' empyrical/tests/test_perf_attrib.py
```

Comment:
Can you try to simplify this a bit? It is quiet hard to grasp.

Reply:
I'd love to, but that's the simplest I could think of unfortunately... The tests are using a custom script, so `pytestCheckHook` and `pytestFlagsArray` won't help, so I'm patching the test file and disable the failing test with `@unittest.skip`. Any suggestion is welcome!

Reply:
I swapped it for `substituteInPlace`, it's slightly easier to read I believe